btk/metrics.py

```diff
@@ -163,13 +164,13 @@ def get_detection_match(
 
     """
     if "x_peak" not in true_table.colnames:
-        raise KeyError("True table has no column x_peak")
+        raise KeyError("True table has no column ra")
     if "y_peak" not in true_table.colnames:
-        raise KeyError("True table has no column y_peak")
+        raise KeyError("True table has no column dec")
     if "x_peak" not in true_table.colnames:
-        raise KeyError("Detection table has no column x_peak")
+        raise KeyError("Detection table has no column ra")
     if "y_peak" not in true_table.colnames:
```

Comment:
is this a typo? did you mean to change the str in the if statement too?

Reply:
Typo, I changed it and then changed it back and forgot to set it right again.